{
    "id": 206994,
    "name": "CVE-2022-4133",
    "cwe": "Unknown",
    "modified": "2023-11-07T02:57:01",
    "published": "2023-02-11T00:23:26",
    "status": "Rejected",
    "summary": "Rejected reason: We were unable to verify this vulnerbility.",
    "cvss3": null,
    "cvss2": null,
    "cpes": []
}